kuma/wiki/models.py

```diff
@@ -1743,11 +1743,8 @@ def previous(self):
         Returns the previous approved revision or None.
         """
         try:
-            return self.document.revisions.filter(
-                is_approved=True,
-                created__lt=self.created,
-            ).order_by('-created')[0]
-        except IndexError:
+            return self.get_previous_by_created(is_approved=True)
+        except self.DoesNotExist:
             return None
```

Comment:
Yeah, I think the comments were wrong or something. fscholz may remember better what https://bugzilla.mozilla.org/show_bug.cgi?id=1174820 included